composer.json

```diff
@@ -15,9 +15,15 @@
             "homepage": "https://github.com/jackalope/jackalope-jackrabbit/contributors"
         }
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/dbu/dbal.git"
+        }
+    ],
     "require": {
         "php": ">=5.3.3",
-        "doctrine/dbal": "~2.5,>=2.5.0",
+        "doctrine/dbal": "dev-backport-creation-fix-to-2.4 as 2.4.99",
```

Comment:
i will remove this once dbal has merged things.

but as far as i can see, all is fine with the name. i stopped all builds but one to see quicker whether things work out.